docs/en/basic/Command-Line-Interface.md

```diff
@@ -163,6 +163,31 @@ The `runTests` command runs end-to-end tests on an Alluxio cluster to provide a
 ./bin/alluxio runTests
 ```
 
+### runUfsTests
+
+The `runUfsTests` aims to test the integration between Alluxio and the given ufs. The ufs tests
+contain all the Alluxio semantics with the ufs.
+
+`--help` will provide the detail guidance on how to run this command.
+`--path <ufs_path>` is required to provide the full ufs path to run ufs tests against.
+
+The usage of this command includes:
+* Test if the given ufs credentials are valid before having a running Alluxio cluster and mounting the ufs to Alluxio.
+* If the given Ufs is S3, this test can also be used as a S3 compatibility test to test if the target under filesystem can
+  fulfill the minimum S3 compatibility requirements in order to work well with Alluxio through Alluxio's integration with S3.
+* Test the integration between Alluxio and the given ufs is not broken. This usage is more related to Alluxio developers. 
```

Comment:
I would reword this entire bullet "Validate the contract between Alluxio and the given UFS. This is primarily intended for Alluxio developers. Developers are required to add test coverage for changes to an Alluxio UFS module and run those tests to validate."

Reply:
Changed the entire bullet. Thanks! The docs are much easier to understand.